consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/IbftEventQueue.java

```diff
@@ -26,16 +26,20 @@
 public class IbftEventQueue {
   private final BlockingQueue<IbftEvent> queue = new LinkedBlockingQueue<>();
 
-  private static final int MAX_QUEUE_SIZE = 1000;
   private static final Logger LOG = LogManager.getLogger();
+  private final int maxQueueSize;
```

Comment:
What happened to using `queueLimit`?

Reply:
It's still there. I missed updating the existing field name in the IbftEventQueue.

Reply:
updated to make it consistent with the new config names